common/persistence/persistenceInterface.go

```diff
@@ -215,6 +216,7 @@ type (
 	InternalGetOrCreateShardRequest struct {
 		ShardID         int32
 		CreateShardInfo func() (rangeID int64, shardInfo *commonpb.DataBlob, err error)
```

Comment:
how about `CreateShardInfo func() (rangeID int64, shardInfo *commonpb.DataBlob, err error)` -> default shard info if persistence decides to create?

e.g. 
caller: 
GetOrCreate(shardID int64, defaultShardInfo *commonpb.DataBlob) (*commonpb.DataBlob, error)

Reply:
keeping this for now